docusaurus/sidebars-ios.json

```diff
@@ -1,49 +1,453 @@
-      "home",
-      "introduction",
-      {
-        "Guides": [
-          "guides/integration",
-          "guides/getting-started",
-          "guides/working-with-channel-list",
-          "guides/working-with-a-channel",
-          "guides/working-with-messages",
-          "guides/working-with-user",
-          "guides/pinned-messages",
-          "guides/working-with-attachments",
-          "guides/working-with-reactions",
-          "guides/working-with-members",
-          "guides/working-with-watchers",
-          "guides/events",
-          "guides/push-notifications",
-          "guides/connection-status",
-          "guides/ui-customization",
-          "guides/moderation",
-          "guides/filter-query-guide",
-          "guides/multi-tenancy",
-          "guides/debugging"
-        ]
-      },
-      { "StreamChat Controllers":
-        [
-          {
-           "type": "autogenerated",
-           "dirName": "controllers"
-          }
-         ] 
-       },
-      
-      { "UI Components": 
-        [
-         {
-          "type": "autogenerated",
-          "dirName": "ui-components"
-         }
-        ] 
-      },
-      "migrating-from-1.x-and-2.x",
-      "faq",
-      "glossary"
-    ]
-  }
+  "docs": [
+    "home",
+    "introduction",
+    {
+      "Guides": [
+        {
+          "Getting Started": [
+            "guides/getting-started",
+            "guides/integration"
+          ]
+        },
+        {
+          "General": [
+            "guides/ui-customization",
+            "guides/working-with-user",
+            "guides/events",
+            "guides/push-notifications",
+            "guides/connection-status",
+            "guides/moderation",
+            "guides/multi-tenancy"
+          ]
+        },
+        {
+          "Working with ChannelList": [
+            "guides/working-with-channel-list",
+            "guides/working-with-a-channel",
+            "guides/working-with-members",
+            "guides/working-with-watchers",
+            "guides/filter-query-guide"
+          ]
+        },
+        {
+          "Working with MessageList": [
+            "guides/working-with-messages",
+            "guides/pinned-messages",
+            "guides/working-with-attachments",
+            "guides/working-with-reactions"
+          ]
+        },
+        "guides/debugging"
+      ]
+    },
+    {
+      "StreamChat Controllers": [
+        "controllers/controllers-overview",
+        "ReferenceDocs/Sources/StreamChat/Controllers/Controller",
+        "ReferenceDocs/Sources/StreamChat/Controllers/EntityChange",
+        "ReferenceDocs/Sources/StreamChat/Controllers/ListChange",
+        {
+          "DataController": [
+            "ReferenceDocs/Sources/StreamChat/Controllers/DataController",
+            "ReferenceDocs/Sources/StreamChat/Controllers/DataController.State",
+            "ReferenceDocs/Sources/StreamChat/Controllers/DataControllerStateDelegate"
+          ]
+        },
```

Comment:
These were to be at the bottom of this group

Reply:
Done.